modules/apps/headless/headless-builder/headless-builder-test/src/testFunctional/macros/APIBuilderUI.macro

```diff
@@ -60,6 +60,19 @@ definition {
 			locator1 = "Button#ANY");
 	}
 
+	macro goToDeleteAPIApplication {
```

Comment:
Maybe it's better to name it like deleteAPIApplication?

Reply:
deleteAPIApplication we has the same name in ApplicationAPI.macro